src/device-state.ts

```diff
@@ -481,11 +481,7 @@ export async function setTarget(target: TargetState, localSource?: boolean) {
 
 	globalEventBus.getInstance().emit('targetStateChanged', target);
 
-	const { uuid, apiEndpoint } = await config.getMany([
-		'uuid',
-		'apiEndpoint',
-		'name',
-	]);
+	const { uuid, apiEndpoint } = await config.getMany(['uuid', 'apiEndpoint']);
```

Comment:
This change wasn't really necessary, but the name query was being done without any reason